std/range/primitives.d

```diff
@@ -944,9 +962,9 @@ unittest
     {
         @property bool empty();
         @property D save();
-        @property int front();
+        int front();
         void popFront();
-        @property int back();
+        int back();
         void popBack();
         ref int opIndex(uint);
         @property size_t length();
```

Comment:
I'd suggest leaving `D` with everything being `@property` like it is now, and adding `E` with `@property` on none of them just to make sure that we don't have any weirdness where some of them are required to be `@property` and some aren't, and it's not caught, because the test for the type that _is_ a random access range doesn't test both cases.